docs/whats-new.asciidoc

```diff
@@ -4,172 +4,36 @@
-
-[float]
-=== View {es} queries that run during rule execution
-
-When previewing a rule, you can also {security-guide}/rules-ui-create.html#view-rule-es-queries[learn about its {es} queries], which are submitted when the rule runs. This information can help you identify and troubleshoot potential rule issues. You can also use it to confirm that your rule is retrieving the expected data. This option is provided for {esql} and EQL rules only.
-
-[float]
-=== Alert suppression is generally available for more rule types
-
-{security-guide}/alert-suppression.html[Alert suppression] is generally available for the indicator match, threshold, {ml}, {esql}, and new terms rule types. It is still in technical preview for event correlation rules.
-
-[float]
-== Investigations enhancements
-
-[float]
-=== Add notes to alerts, events, and Timelines
-
-You can now attach {security-guide}/add-manage-notes.html[notes] to alerts, events, and Timelines, and manage them from the **Notes** page. This provides an easy way to incorporate notes into your investigative workflows to coordinate responses, conduct threat hunting, and share investigative findings.
-
-[role="screenshot"]
-image::whats-new/images/8.16/new-note-alert-event.png[New note added to an alert]
-
-[float]
-=== View analyzed events from the alert details flyout
-
-preview:[] By enabling the new `securitySolution:enableVisualizationsInFlyout` advanced setting, you can {security-guide}/view-alert-details.html#expanded-visualizations-view[view analyzed alerts and events] in the **Visualize** tab of the alert details flyout. This allows you to maintain the context of the Alerts table during your investigation and provides an easy way to preview related alerts and events.
-
-[role="screenshot"]
-image::whats-new/images/8.16/visualize-tab-lp-alert-details.gif[Examine alert details from event analyzer, 80%]
-
-[float]
-=== Resize alert and event details flyouts
-
-You can now resize the alert and event details flyouts and choose how they're displayed—over the Alerts table or next to it.
-
-[role="screenshot"]
-image::whats-new/images/8.16/flyout-settings.gif[Change alert details flyout settings]
-
-[float]
-== {elastic-defend} and response actions enhancements
-
-[float]
-=== More SentinelOne third-party response actions
+=== Logsdb index mode with detection rules and alerts
 
-Additional third-party response actions are available using Elastic's {security-guide}/third-party-actions.html#sentinelone-response-actions[SentinelOne] integration and connector:
+The {ref}/logs-data-stream.html[logsdb index mode] allows you to store log data more efficiently. If you're considering using it, refer to Using logsdb index mode with {elastic-sec} to learn how it can impact your rules and alerts.
```

Comment:
Re-adding the link so you can merge this comment after I merge https://github.com/elastic/security-docs/pull/6272.

```suggestion
The {ref}/logs-data-stream.html[logsdb index mode] allows you to store log data more efficiently. If you're considering using it, refer to <<detections-logsdb-index-mode-impact,Using logsdb index mode with {elastic-sec}>> to learn how it can impact your rules and alerts.

NOTE: To use the {ref}/mapping-source-field.html#synthetic-source[synthetic `_source`] feature, you must have the appropriate subscription. Refer to the subscription page for https://www.elastic.co/subscriptions/cloud[Elastic Cloud] and {subscriptions}[Elastic Stack/self-managed] for the breakdown of available features and their associated subscription tiers.
```